data/brands/amenity/atm.json

```diff
@@ -240,6 +240,70 @@
         "name:he": "בנק טינקוף",
         "name:ru": "Тинькофф Банк"
       }
+    },
+    {
+        "displayName": "高雄銀行",
+        "id": "bankofkaohsiung-9a6797",
+        "locationSet": {"include": ["tw"]},
+        "tags": {
+          "amenity":"atm",
+          "brand":"高雄銀行",
+          "brand:en":"Bank of Kaohsiung",
+          "brand:wikidata":"Q11673248",
+          "brand:zh":"高雄銀行",
+          "name":"高雄銀行",
+          "name:en":"Bank of Kaohsiung",
+          "name:nanKo":"hiông Gîn-hâng",
+          "name:nan:HJ": "高雄銀行",
+          "name:nan:POJ":"Ko-hiông Gîn-hâng",
+          "name:nan:TL":"Ko-hiông Gîn-hâng",
+          "name:zh":"高雄銀行",
+          "operator":"雄銀行",         
+          "operator:wikidata":"Q11673248",
+          "matchNames":"bok,高銀,高雄市銀行,高雄銀"
```

Comment:
`matchNames` should go outside of tags, see https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional